boards/arm/thingy91_nrf9160/thingy91_nrf9160.yaml

```diff
@@ -1,5 +1,5 @@
-identifier: nrf9160_pca20035
-name: nRF9160-PCA20035
+identifier: thingy91_nrf9160
+name: THINGY91-nRF9160
```

Comment:
Suggest `name: Thingy91-NRF9160` To align with Thingy:52. Or actually, `name: Thingy91-nRF9160` looks slightly better..